tests/unit/test_parsers.py

```diff
@@ -363,41 +363,6 @@
             Path(dir_path, "data", "sparkle", "sparkle1_html_parser_result.json"),
         ),
         # Telstra
-        (
-            HtmlParserTelstra1,
```

Comment:
Hi @unpapillon , you should not remove previous tests that grant that the previous format is still supported.

If the new parser is incompatible with the ones before, you could implement a different processor in parallel, as done for other providers (if the first fails to parse, it tries with the second):
https://github.com/networktocode/circuit-maintenance-parser/blob/develop/circuit_maintenance_parser/provider.py#L206
https://github.com/networktocode/circuit-maintenance-parser/blob/develop/circuit_maintenance_parser/provider.py#L216

Reply:
okay I'll see what I can do